velox/expression/ControlExpr.cpp

```diff
@@ -66,6 +66,15 @@ void ConstantExpr::evalSpecialFormSimplified(
   }
 }
 
+std::string ConstantExpr::toString() const {
+  if (sharedSubexprValues_ == nullptr) {
+    return fmt::format("{}:{}", value_.toJson(), type()->toString());
```

Comment:
Should we use a double-colon between the value and the type? I feel that will make it more readable.

Reply:
I'm using single column to match Presto:

```
- TopN[10000 by (returnflag ASC_NULLS_LAST, linestatus ASC_NULLS_LAST)] => [returnflag:varchar(1), linestatus:varchar(1), sum:double, sum_8:double, sum_9:double, sum_10:double, avg:double, avg_11:double, avg_12:double, count:bigint]
```